### PR TITLE
ROCm Version page

### PR DESCRIPTION
**Pull request by samjwu**
https://advanced-micro-devices-demo--2331.com.readthedocs.build/en/2331/release/versions.html

### CODE REVIEW

File: docs/release/versions.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+| [5.3.0](https://rocm.docs.amd.com/en/docs-5.3.0/) | Oct 4, 2022 |
+| [5.2.3](https://rocm.docs.amd.com/en/docs-5.2.3/) | Aug 18, 2022 |
+| [5.2.1](https://rocm.docs.amd.com/en/docs-5.2.1/) | Jul 21, 2022 |
+| [5.2.0](https://rocm.docs.amd.com/en/docs-5.2.0/) | Jun 28, 2022 |
+| [5.1.3](https://rocm.docs.amd.com/en/docs-5.1.3/) | May 20, 2022 |
+| [5.1.1](https://rocm.docs.amd.com/en/docs-5.1.1/) | Apr 8, 2022 |
+| [5.1.0](https://rocm.docs.amd.com/en/docs-5.1.0/) | Mar 30, 2022 |
+| [5.0.2](https://rocm.docs.amd.com/en/docs-5.0.2/) | Mar 4, 2022 |
+| [5.0.1](https://rocm.docs.amd.com/en/docs-5.0.1/) | Feb 16, 2022 |
+| [5.0.0](https://rocm.docs.amd.com/en/docs-5.0.0/) | Feb 9, 2022 |
```
Comment (saadrahim) [inline docs/release/versions.md:23]: Years are still off, 2021 or 2020 should be starting year.

**Review by saadrahim — CHANGES_REQUESTED** [review]

File: docs/release/versions.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# ROCm Versions
+
+| Version | Release Date |
+| [5.6.0](https://rocm.docs.amd.com/en/docs-5.6.0/) | Jun 28, 2022 |
```
[the remark this answers is not in the document]
Comment (samjwu) [reply docs/release/versions.md]: Fixed to match https://github.com/RadeonOpenCompute/ROCm/tags

File: docs/release/versions.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# ROCm Versions
```
[the remark this answers is not in the document]
Comment (samjwu) [reply docs/release/versions.md]: Done

File: docs/release/versions.md (hunk 1 above)
Comment (samjwu) [reply docs/release/versions.md:23]: Fixed to match https://github.com/RadeonOpenCompute/ROCm/tags

**Review by saadrahim — APPROVED** [review]

**Pull request merged**